bom/runtime/pom.xml

```diff
@@ -213,6 +213,15 @@
                 <scope>import</scope>
             </dependency>
 
+            <!-- Vert.X dependencies, imported as a BOM -->
+            <dependency>
+                <groupId>io.vertx</groupId>
+                <artifactId>vertx-stack-depchain</artifactId>
+                <version>${vertx.version}</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
```

Comment:
@rsvoboda could you check that adding this BOM doesn't create dependency issues? Especially for Netty and Jackson as the Vert.x BOM also includes them.

Thanks!

Reply:
will do ;) later today or tomorrow